connectors/flink/src/main/java/io/delta/flink/internal/table/DeltaCatalog.java

```diff
@@ -218,7 +217,7 @@ public void createTable(DeltaCatalogBaseTable catalogTable, boolean ignoreIfExis
             }
 
             // Add table to metastore
-            DeltaMetastoreTable metastoreTable =
+            ResolvedCatalogTable metastoreTable =
```

Comment:
needs to be a `ResolvedCatalogTable` since https://github.com/apache/flink/commit/3d350485755e47d2b09ac1b2067b6119ef960b5a